charts/stable/aria2/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+appVersion: 1.0.0
```

Comment:
This should be the application version

Reply:
upsteam doesnt use semver, attempted "Semvered" date based version